lago/cmd.py

```diff
@@ -154,24 +162,17 @@ def do_init(
                 template_repo_path
             )
         else:
-            try:
-                repo_name = (
-                    template_repo_name
-                    or lago.config.get('template_default_repo')
```

Comment:
I see no evidence of 'template_default_repo' parameter in our code-base, of course could be that users used it. You know if we actually used it anywhere?

As a side note, currently ```generate-config```  filters CLI arguments that don't have default values, such as template_repo_name(you can still add it there or in ENV), but maybe a better approach would be to leave them commented out, so their existence won't be forgotten.